<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroWindow</class>
 <widget class="QMainWindow" name="RegistroWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de Usuario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="labelTitulo">
      <property name="text">
       <string>Registro de nuevo usuario</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineNombre">
      <property name="placeholderText">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineCorreo">
      <property name="placeholderText">
       <string>Correo electrónico</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineContrasena">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Contraseña</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnRegistrar">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Registrarse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

